src/app/state/staking/saga.ts

```diff
@@ -85,7 +85,14 @@ function* loadDebondingDelegations(publicKey: Uint8Array) {
 
 function* refreshValidators() {
   const { accounts } = yield* call(getExplorerAPIs)
-  const validators = yield* call(() => accounts.getValidatorsList({ limit: 500 }))
```

Comment:
cc @lukaw3d on the style change from using a lambda back to `call` with context and args

Reply:
Ah, fix in https://github.com/oasisprotocol/oasis-wallet-web/pull/442